cogs/welcome/welcome.py

```diff
@@ -28,8 +31,22 @@ class Welcome(commands.Cog):  # pylint: disable=too-many-instance-attributes
     def __init__(self, bot: Red):
         self.bot = bot
         self.config = Config.get_conf(self, identifier=5842647, force_registration=True)
+
         self.config.register_guild(**DEFAULT_GUILD)
 
+        self.data_dir = data_manager.cog_data_path(cog_instance=self)
+        self.img_dir = os.path.join(self.data_dir, "welcome_imgs")
```

Comment:
Since [`data_manager.cog_data_path()`](https://docs.discord.red/en/stable/framework_datamanager.html#redbot.core.data_manager.cog_data_path) returns a [`Path`](https://docs.python.org/3/library/pathlib.html#pathlib.Path), consider favoring [`Path`](https://docs.python.org/3/library/pathlib.html#pathlib.Path) (i.e., the [slash operator](https://docs.python.org/3/library/pathlib.html#operators)) over [`os.path`](https://docs.python.org/3/library/os.path.html) methods.
Also do create a constant instead of using a literal string here.
```suggestion
        self.img_dir = self.data_dir / WELCOME_IMG_FOLDER
```